### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by leon-andria**
- use correct split config options with quantum

### CODE REVIEW

File: modules/fuse-box-responsive-api/index.js [hunk 1]
```diff
@@ -235,7 +235,17 @@
             /* @if codeSplitting */
             if (bMapping.i && bMapping.i[id]) {
                 var data = bMapping.i[id];
-                req(bMapping.c.b + data[0], function(err, result) {
+                /* @if universal */
+                const contextualPath = isBrowser ? bMapping.c.b : bMapping.c.s;
```
Comment (nchanged) [inline modules/fuse-box-responsive-api/index.js]: LGTM except for `const` please, convert it to `var` and I think we can merge it.
Good job!

Comment (nchanged) [reply modules/fuse-box-responsive-api/index.js]: Oh there is one thing. `bMapping.c.b` can it be undefined ?! Can you check that please

**Comment by leon-andria** [comment]
It cant be undefined its a config that you auto generate with default value. The moment you split, it's there. That's make sense.

**Comment by nchanged** [comment]
Ok, then tend to my other commit, with replacing const to var

**Comment by leon-andria** [comment]
The changes was done.

**Comment by nchanged** [comment]
Lemme dwell on it a bit, i will merge it today

**Comment by nchanged** [comment]
Why isn't the commit in the format of (type): message? 
Did you do npm install beforehand? it shouldn't have allowed you to commit like this

**Pull request merged**